Mathlib/NumberTheory/FLT/Basic.lean

```diff
@@ -0,0 +1,96 @@
+/-
+Copyright (c) 2023 Kevin Buzzard. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Buzzard, Yaël Dillies
+-/
+import Mathlib.Data.Nat.Parity
+import Mathlib.Data.Rat.Defs
+import Mathlib.Tactic.Positivity
+import Mathlib.Tactic.TFAE
+
+/-!
+# Statement of Fermat's Last Theorem
+
+This file states the Fermat Last Theorem. We provide a statement over a general semiring with
```

Comment:
This grammar seems very strange to me.

Reply:
What would you want? Something like this?
```suggestion
This file states Fermat's Last Theorem. We provide a statement over a general semiring with
```
Feel free to commit the suggestion if so.